packages/framework/ai-collab/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "@fluid-experimental/ai-collab",
+	"name": "@fluidframework/ai-collab",
```

Comment:
We'll need to update the `exports` section here to export alpha APIs via `/alpha`.